src/Service/Google.php

```diff
@@ -23,7 +23,7 @@
  */
 class Google extends Service
 {
-    const URL = 'http://finance.google.com/finance/converter?a=1&from=%s&to=%s';
+    const URL = 'https://www.google.es/search?q=1+%s+to+%s';
```

Comment:
Could we use google.com please ?

Reply:
Up. Since I also use your package in production once an hour for a few currencies, I'll try my best to detect any important change in any of these rates. So, I'll add an extra check in my code to detect a variation (let's say) -+10%, to be as sure as possible everything works fine.